devicetypes/smartthings/zwave-basic-window-shade.src/zwave-basic-window-shade.groovy

```diff
@@ -58,11 +58,25 @@ metadata {
 				defaultValue: false,
 				displayDuringSetup: false
 			)
+            
+                input(title: "Aeotec Nano Shutter settings",
```

Comment:
Please, fix spacing: 4 spaces -> 1 tab (applies to the whole file)

Reply:
@KKlimczukS - Fixed spacing